content/spire/docs/configuring.md

```diff
@@ -214,6 +214,19 @@ The Unix Workload Attestor works by determining kernel metadata from the workloa
 
 For more information, including details of the exposed selectors, refer to the corresponding SPIRE documentation for the [Unix Workload Attestor plugin](https://github.com/spiffe/spire/blob/master/doc/plugin_agent_workloadattestor_unix.md).
 
+# Configuring where to store agent and server data
+_This configuration applies to the SPIRE Server and SPIRE Agent_
+
+The `data_dir` option in the `agent.conf` and `server.conf` configuration files sets the directory for SPIRE runtime data.
+
+If you specify a relative path for `data_dir` by starting the path with `./` then `data_dir` is evaluated based on the current working directory from which you run the SPIRE Agent or Server. Using a relative path for `data_dir` can be useful for initial evaluation of SPIRE, but for production deployments you may want to set `data_dir` to an absolute path. By convention, specify `"/opt/spire/data"` for `data_dir` if you have installed SPIRE in `/opt/spire`.
```

Comment:
"evaluated based on the current working directory from which you run the SPIRE Agent or Server" is a little ambiguous. Consider "evaluated based on the current working directory from which you executed the `spire-agent` or`spire-server` command"

Reply:
Good suggestion, done.